### PR TITLE
Add Annotate gem and notes

### PR DESCRIPTION
**Pull request by innatewonderer**
I find it useful to be able to reference what attributes are still applicable to each model.

### CODE REVIEW

**Comment by sudara** [comment]
How would you feel about putting the annotation at the bottom of the files for reference? I kind of enjoy being free from the clutter when I first open a file. I could maybe get used to it again if you prefer them up top...

**Comment by innatewonderer** [comment]
I have no preference, as long as I can see the model's attributes. Let me see if there is a setting somewhere.

**Comment by sudara** [comment]
Ok cool! One of the reason I originally removed them is they would get out of sync with the schema. Since I work on many apps where this is the case, I essentially just refer to the schema.rb to assess the attributes. This will be nice at the bottom if we can keep it up to date!

**Comment by innatewonderer** [comment]
This should help I think:
```
To automatically annotate every time you run db:migrate, either run rails g annotate:install or add Annotate.load_tasks to your `Rakefile`. See the configuration in Rails section for more info.
```

**Comment by sudara** [comment]
The rake task sounds like a great way to solve it!

File: config/environments/development.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Rails.application.configure do
   # Verifies that versions and hashed value of the package contents in the project's package.json
-    config.webpacker.check_yarn_integrity = true
-    config.webpacker.check_yarn_integrity = true  # Settings specified here will take precedence over those in config/application.rb.
+    config.webpacker.check_yarn_integrity = false
+    config.webpacker.check_yarn_integrity = false  # Settings specified here will take precedence over those in config/application.rb.
```
Comment (sudara) [inline config/environments/development.rb]: Are these changes related? I think we want this to be set back to `true` so that yarn complains when package.json changes. Not sure why there were two lines though, maybe we can delete one?

File: app/models/track.rb [hunk 2]
```diff
@@ -1,14 +1,22 @@
...
+#  playlist_id :integer
+#  user_id     :integer
+#
+# Indexes
+#
+#  index_tracks_on_asset_id     (asset_id)
+#  index_tracks_on_is_favorite  (is_favorite)
+#  index_tracks_on_playlist_id  (playlist_id)
+#  index_tracks_on_user_id      (user_id)
 #
```
Comment (sudara) [inline app/models/track.rb]: Hah, like this file had very very old leftovers in the header, would you mind moving it to the footer?

File: lib/tasks/auto_annotate_models.rake [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+      'force'                     => 'false',
+      'classified_sort'           => 'true',
+      'trace'                     => 'false',
+      'wrapper_open'              => nil,
+      'wrapper_close'             => nil,
+      'with_comment'              => true
+    )
+  end
+
+  Annotate.load_tasks
```
Comment (innatewonderer) [inline lib/tasks/auto_annotate_models.rake]: This should run annotate automagically on `rake db:migrate`. I'll test to confirm

**Comment by innatewonderer** [comment]
@sudara mergeable
